ivy/functional/frontends/jax/__init__.py

```diff
@@ -15,6 +15,7 @@
 from . import _src
 from ._src import tree_util
 
+__version__ = ivy.backend_version.get("version", "version not available!")
 
```

Comment:
The version of the frontend is not necessarily same as the corresponding backend (I think that's what you are doing here). For example, we could use an old pytorch version as frontend and transpile it to a newer pytorch version, in which case frontend and backend would be torch but their versions would differ.
I think we need to get the frontend version from here https://github.com/unifyai/ivy/blob/b54f96889e45c45f22eee75dae79af8b518faa32/ivy/functional/frontends/__init__.py#L88

Reply:
No, I don't think so. We need to get the backend version here because all the computations being performed rely on this compatibility.